src/item/LingeringPotion.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+ */
+
+declare(strict_types=1);
+
+namespace pocketmine\item;
+
+use pocketmine\data\runtime\RuntimeDataReader;
+use pocketmine\data\runtime\RuntimeDataWriter;
+use pocketmine\entity\Location;
+use pocketmine\entity\projectile\SplashPotion as SplashPotionEntity;
+use pocketmine\entity\projectile\Throwable;
+use pocketmine\player\Player;
+
+class LingeringPotion extends ProjectileItem{
+
+	private PotionType $potionType;
+
+	public function __construct(ItemIdentifier $identifier, string $name){
+		$this->potionType = PotionType::WATER;
+		parent::__construct($identifier, $name);
+	}
+
+	protected function describeType(RuntimeDataReader|RuntimeDataWriter $w) : void{
```

Comment:
This should be `RuntimeDataDescriber`, we're not in PM5 alpha anymore

See https://github.com/pmmp/PocketMine-MP/blob/49da50659f08472ddc41b6925098006ba95de3f5/src/item/SplashPotion.php#L36-L38